docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -90,18 +105,17 @@ Here is a fully working example:
 
 ## Toggling color mode
 
-You can use the React context to toggle the mode with a button inside your page.
+If you want to provide your users a way to toggle between modes, you can use React's context adding it to a button's `onClick` event:
 
 {{"demo": "ToggleColorMode.js", "defaultCodeOpen": false}}
 
 ## System preference
 
-Users might have specified a preference for a light or dark theme.
-The method by which the user expresses their preference can vary. It might be a system-wide setting exposed by the Operating System, or a setting controlled by the User Agent.
+The method by which users express their mode preference can vary. It might be a system-wide setting exposed by the Operating System, or a setting controlled by the User Agent (user specifying a preference for a light or dark theme).
```

Comment:
This is very wordy, and I think it can be pared down quite a bit. Is the end clause ("either systemwide, or for a single user agent") really a necessary detail? What I mean by that is: does the implementation change depending on how the preference is defined? Or is the same either way? If it's the same, I think we can omit that detail.

```suggestion
Users might have a preference for light or dark mode that they've set through their operating system—either systemwide, or for a single user agent.
```